newdle/client/src/components/Home.js

```diff
@@ -2,21 +2,73 @@ import React from 'react';
 import {Button} from 'semantic-ui-react';
 import LoginRequired from './LoginRequired';
 import {useRouter} from '../util/router';
+import {Icon} from 'semantic-ui-react';
+
+import {ReactComponent as TeamIcon} from '../images/team.svg';
+import {ReactComponent as TimetableIcon} from '../images/timetable.svg';
+import {ReactComponent as DiscussIcon} from '../images/discuss-issue.svg';
+import {ReactComponent as GitHubIcon} from '../images/github.svg';
+
+import styles from './Home.module.scss';
 
 export default function Home() {
   const router = useRouter();
 
   return (
-    <>
-      <h2>Home</h2>
-      <LoginRequired
-        component={Button}
-        onClick={() => {
-          router.history.push('/new');
-        }}
-      >
-        Create newdle
-      </LoginRequired>
-    </>
+    <div>
+      <div className={styles.container}>
+        <h2>How does it work?</h2>
+        <ol className={styles.box}>
+          <li>
+            <TeamIcon className={styles.icon} />
+            Choose your participants
+          </li>
+          <li>
+            <TimetableIcon className={styles.icon} />
+            Set the time slots based on their availability
+          </li>
+          <li>
+            <DiscussIcon className={styles.icon} />
+            <strong>newdle</strong> will collect the answers!
+          </li>
+        </ol>
+        <div className={styles.button}>
+          <LoginRequired
+            component={Button}
+            color="violet"
+            icon
+            labelPosition="right"
+            size="big"
+            onClick={() => {
+              router.history.push('/new');
+            }}
+          >
+            Get started
+            <Icon name="caret right" />
+          </LoginRequired>
+        </div>
+        <div className={styles.footer}>
+          <p className={styles.opensource}>
+            newdle is Open Source Software
+            <a href="https://github.com/indico/newdle">
+              <GitHubIcon />
+            </a>
+          </p>
+          <p className={styles.cern}>
+            Made at <a href="https://home.cern">CERN</a>, the place where the web was born.
+          </p>
+          <p className={styles.credits}>
+            "Team", "Timetable" and "Discuss Issue" icons made by{' '}
+            <a href="https://www.flaticon.com/authors/freepik" title="Freepik">
+              Freepik
+            </a>{' '}
+            from{' '}
+            <a href="https://www.flaticon.com/" title="Flaticon">
```

Comment:
`target="_blank" rel="noopener noreferrer"`? Generally I don't like opening stuff in a new window but maybe here it's appropriate?

Reply:
I guess we should do the same for the CERN link if we do it here

Reply:
(and GitHub, since we're at it)